contracts/token/ERC721/ERC721.sol

```diff
@@ -249,6 +248,24 @@ contract ERC721 is ERC165, IERC721 {
         _tokenOwner[tokenId] = address(0);
     }
 
+    /**
+     * @dev Internal function to transfer a token ID from the list of a given address to another one.
+     * Note that this function is left internal to make ERC721Enumerable possible, but is not
+     * intended to be called by custom derived contracts: in particular, it emits no Transfer event,
```

Comment:
`ERC20` `_transfer` does emit the `Transfer` event. I'm thinking we should aim for the same here. What do you think? Note that it doesn't clear approval either, so that is consistent.

Reply:
This should be completely analogous to `ERC20` `_transfer` IMO: its usage should be allowed to derived contracts.

Reply:
Are you suggesting implementing a full-fledged `internal` transfer function? The `add` and `remove` ones are not that way because they needed to support both `transfer` and `mint`/`burn`, but we could definitely have `transfer` be 'complete'.

Reply:
Yes.